itinerant-iterators/app/steganography/lsb.py

```diff
@@ -0,0 +1,85 @@
+from .. import byteutils
+from .steganography import Image, Steganography
+
+EOM = "$$$"
+
+
+def pixels_top_left(pixels: list[list[int]], bytes_input: bytes):
+    """A generator that chooses pixels starting from the top left"""
+    for pixel in pixels:
+        yield pixel
+
+
+def pixels_space_evenly(pixels: list[list[int]], bytes_input: bytes):
+    """A generator that chooses pixels such that they are spaced evenly throughout the image"""
+    ...  # TODO
+
+
+class Lsb(Steganography):
```

Comment:
Sometimes, spelling out an acronym in full is better in the longrun.

```suggestion
class LeastSignificantBit(Steganography):
```